pkg/kubelet/server/server.go

```diff
@@ -393,17 +391,15 @@ func (s *Server) InstallDefaultHandlers(enableCAdvisorJSONEndpoints bool) {
 	)
 
 	s.addMetricsBucketMatcher("spec")
-	if enableCAdvisorJSONEndpoints {
-		ws := new(restful.WebService)
-		ws.
-			Path(specPath).
-			Produces(restful.MIME_JSON)
-		ws.Route(ws.GET("").
-			To(s.getSpec).
-			Operation("getSpec").
-			Writes(cadvisorapi.MachineInfo{}))
-		s.restfulCont.Add(ws)
-	}
+	ws := new(restful.WebService)
```

Comment:
This is as interesting as it gets. I get:
```
# k8s.io/kubernetes/pkg/kubelet/server
pkg/kubelet/server/server.go:394:5: no new variables on left side of :=
# k8s.io/kubernetes/pkg/kubelet/server
vet: pkg/kubelet/server/server.go:394:5: no new variables on left side of :=
make[1]: *** [Makefile:388: vet] Error 1
```
and a bunch of other failures are probably related as well.

Interestingly, all I did was revert some part of #78504, where this was exactly like this. Likely I am missing somehting (not sure what though!?)